catalyst-gateway/tests/Earthfile

```diff
@@ -5,7 +5,7 @@ build:
     RUN cargo build -p cat-gateway --release
     SAVE ARTIFACT target/release/cat-gateway cat-gateway
 
-cat-gateway-publish:
+package-cat-gateway:
```

Comment:
Should we move this to the main cat-gateway Earthfile?

Reply:
I'm not sure, I only renamed the target to take advantage of the CI features, the target itself is not touched in this PR.